lldb/include/lldb/API/SBFrameList.h

```diff
@@ -0,0 +1,55 @@
+//===-- SBFrameList.h -------------------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLDB_API_SBFRAMELIST_H
+#define LLDB_API_SBFRAMELIST_H
+
+#include "lldb/API/SBDefines.h"
+
+namespace lldb {
+
+class LLDB_API SBFrameList {
+public:
+  SBFrameList();
+
+  SBFrameList(const lldb::SBFrameList &rhs);
+
+  ~SBFrameList();
+
+  const lldb::SBFrameList &operator=(const lldb::SBFrameList &rhs);
+
+  explicit operator bool() const;
+
+  bool IsValid() const;
+
+  uint32_t GetSize() const;
+
+  lldb::SBFrame GetFrameAtIndex(uint32_t idx) const;
+
+  void Clear();
+
+  void Append(const lldb::SBFrame &frame);
+
+  void Append(const lldb::SBFrameList &frame_list);
+
+  bool GetDescription(lldb::SBStream &description) const;
+
+protected:
+  friend class SBThread;
+
+private:
+  SBFrameList(const lldb::StackFrameListSP &frame_list_sp);
+
+  void SetOpaque(const lldb::StackFrameListSP &frame_list_sp);
```

Comment:
I'd call this SetFrameList, the fact that the ivar is called "m_opaque_sp" is too much of an implementation detail and SetOpaque sounds like it's making the frame list opaque somehow.